Configuration/PyReleaseValidation/test/test_mc_setup/test_MC_setup_reco.sh

```diff
@@ -11,8 +11,16 @@ era=$3
 
 echo '> Running RAW2DIGI,L1Reco,RECO,RECOSIM + PAT steps in ' $release
```

Comment:
Not really about this PR, but this printout is confusing, as it would indicate the RECO step being run in (e.g.) `CMSSW_12_4_*_HLT`, which is not the case. I'd suggest to (either in this PR or in a subsequent one) to either make this to print the release used for RECO, or modify it along it reads the input from HLT ran in `$release`.

Reply:
Let's do it elsewhere.

Reply:
Fixed in the last push.